doc/src/manual/arrays.md

```diff
@@ -1100,6 +1109,42 @@ julia> ([1, 2, 3], [4, 5, 6]) .+ tuple([1, 2, 3])
 ([2, 4, 6], [5, 7, 9])
 ```
 
+
+In all these that have been said, do note that not all operations requires you vectorizing or broadcasting for
+elementwise operations. For some binary operators (e.g `*`, `+`), when used with an array and a scalar, or
+between two arrays, an elementwise operation is *normally* performed. This is important to note because whichever
+style you may choose to use, though the end results will be the same, their execution speeds and allocations may be
+different (depending on the sizes of the arrays).
+
+For example, the `+` operator when used on two arrays of same sizes and dimensions performs elementwise addition,
+just as `.+` would have done:
+
+```jldoctest
+julia> [1, 2] .+ [3, 4]
+2-element Vector{Int64}:
+ 4
+ 6
+
+julia> [1, 2] + [3, 4]
+2-element Vector{Int64}:
+ 4
+ 6
+```
+
+However, calling `@time` on these two operations (performed on a small dataset), we see the difference:
+
+```julia-repl
+julia> a = rand(Float64, 100); b = rand(Float64, 100);
+
+julia> @time a + b;
+  0.000007 seconds (1 allocation: 896 bytes)
+
+julia> @time a .+ b;
+  0.000026 seconds (3 allocations: 960 bytes)
+```
+
+On a large dataset, the time difference may be very small, but the allocations will most likely be different.
+
```

Comment:
IMHO, I don't think it should be removed. I've updated the docs to maybe say explicitly with less words to drive at what I'm trying to say. Maybe you should look at it and see if you think its not useful?

Reply:
I stand by my previous comment.

Reply:
Okay. I will remove as you've said, but before then I would love to ask a question so I understand something clearer.

  Since you're well acquainted with the language, should I worry over which one of these to use when both `a` and `b` are 1D-arrays:

```julia
julia> a + b;

julia> a .+ b;
```

If you say **NO**, why? And if **YES**, why?